setup.py

```diff
@@ -23,7 +23,7 @@
 
 requirements = [
     "qiskit>=0.19.1",
-    "pennylane>=0.8.1",
+    "pennylane>=0.9.0",
```

Comment:
Wondering about the ["unpinned" pair to this in requirements.txt](https://github.com/XanaduAI/pennylane-qiskit/blob/9e6a7f11cb88dbc43c3852fc905103085fbe849b/requirements.txt#L2) :thinking:

Reply:
It should definitely be `pennylane>=0.9` in the requirements.txt as well! We can also likely remove the Networkx pinning, I think?

Reply:
Thanks!

Would there be any specific reason for Networkx? It seems like one would get the correct version with this pinning :thinking: (I can be convinced otherwise).

Reply:
Oh, simply because it echoes what Qiskit is pinning for networkx anyway, so we're just duplicating a downstream dependency. Is there a reason for duplicating the networkx pinning here?

Reply:
Thanks, removed!